libraries/nexus_helper.rb

```diff
@@ -37,5 +37,13 @@ def wait_until_ready!(endpoint, timeout = 15 * 60)
     rescue Timeout::Error
       raise Nexus3NotReady.new(endpoint, nexus_timeout)
     end
+
+    # Load Groovy scripts via the Nexus3 API.
+    class Scripting
+      def self.groovy_script_location(script_name, node)
```

Comment:
You should do a `groovy_script_content` helper :)

Reply:
groovy_content, done.